packages/devtools_app/lib/src/memory/flutter/memory_graph_model.dart

```diff
@@ -62,8 +68,19 @@ Map<LibraryClass, Predefined> predefinedClasses = {
   predefinedString: const Predefined('String', true),
   predefinedList: const Predefined('List', false),
   predefinedMap: const Predefined('Map', false),
+  predefinedHashMap: const Predefined('HashMap', false),
 };
 
+bool isMap(HeapGraphClassLive live) =>
+    live.fullQualifiedName == predefinedMap ||
+    live.fullQualifiedName == predefinedHashMap;
+
+bool isHashMap(HeapGraphClassLive live) =>
+    live.fullQualifiedName == predefinedHashMap;
+
+bool isList(HeapGraphClassLive live) =>
```

Comment:
maybe rename to make clear that this is the built in list class only?

Reply:
done for all list, map, and hashmap.